src/frontend/check_annotation.nit

```diff
@@ -84,6 +84,7 @@ readonly
 writable
 autoinit
```

Comment:
If lateinit replaces autoinit, why is it still in chack_annotation, shouldn't it be removed ?